public/script.js

```diff
@@ -1443,7 +1446,7 @@ function getHiddenBlock(hidden) {
     const hiddenBlock = `
     <div class="text_block hidden_block">
         <small>
-            <p>${hidden} ${hidden > 1 ? 'characters' : 'character'} hidden.</p>
+            <p>` + t`${hidden} ${hidden > 1 ? 'characters' : 'character'} hidden.` + `</p>
```

Comment:
We need to split this. Even though the `${hidden}` variable will be correctly inserted, you won't translate `character`/`characters` with this. I think without too much overhead, we should do it like this:
```suggestion
            <p>` + (hidden > 1 ? t`${hidden} characters hidden.` : t`${hidden} character hidden.`) + `</p>
```